targets/TARGET_STM/TARGET_STM32WL/TARGET_STM32WLE4x8/TOOLCHAIN_ARM/stm32wle4x8.sct

```diff
@@ -0,0 +1,57 @@
+#! armcc -E
```

Comment:
```suggestion
#! armclang -E --target=arm-arm-none-eabi -x c -mcpu=cortex-<CORE_TYPE>
```

Please make sure the shebang in all scatter files specify `armclang` rather than `armcc`.🙌

Reply:
Ack
@JeanMarcR see patch on our internal branch

Reply:
Done